Formula/zrok.rb

```diff
@@ -0,0 +1,40 @@
+require "language/node"
+
+class Zrok < Formula
+  desc "Geo-scale, next-generation sharing platform built on top of OpenZiti"
+  homepage "https://zrok.io"
+  url "https://github.com/openziti/zrok.git",
+      tag:      "v0.3.5",
+      revision: "917891e9b1fbe28e112ff7e399a92d60080c79e4"
```

Comment:
only reason not using tarball file rather than git checkout

Reply:
We weren't intending to add bottles until further on in development. Sorry if that's required, while reading through it didn't appear to be so.

Reply:
Why not use 
```rb
  url "https://github.com/openziti/zrok/archive/refs/tags/v0.3.6.tar.gz"
```

Reply:
Homebrew core formulae must always be build from source. The question is only why you're declaring a git checkout instead of just the download for the tarbal.

Reply:
Ok, sorry I did misunderstand you.

Realistically the choice was just because it seemed more explicit at the time. When running through the original workings on getting the builds through homebrew I had misinterpreted that tarball download to be the build artifacts, not the repo itself. So I opted to use git at the time, and decided kept it.

Reply:
Please switch to tarbal then. Unless you need the git information there are no benefits to this approach

Reply:
Sorry, just now looking back at this. Updated to use the tarball rather than git